package.json

```diff
@@ -32,7 +32,10 @@
     "style-loader": "^0.13.0"
   },
   "dependencies": {
-    "lodash": "^4.1.0",
-    "machina": "^1.1.2"
+    "jquery": "^2.2.0",
+    "jsdom": "^7.2.2",
+    "machina": "^1.1.2",
+    "webpack": "^1.12.11",
```

Comment:
Can I remove the dependencies just by deleting the line safely?
I believe that the same holds true for jsdom